src/api/slice/write_to_slice.rs

```diff
@@ -76,12 +76,12 @@ macro_rules! impl_slice_write_to_slice {
         test_if!{
             $test_tt:
             interpolate_idents! {
-                mod [$id _slice_write_to_slice] {
+                pub mod [$id _slice_write_to_slice] {
                     use super::*;
                     use iter::Iterator;
 
-                    #[test]
-                    fn write_to_slice_unaligned() {
+                    #[cfg_attr(not(target_arch = "wasm32"), test)] #[cfg_attr(target_arch = "wasm32", wasm_bindgen_test)]
+                    pub fn write_to_slice_unaligned() {
```

Comment:
While the modules need to be publich (a [known bug](https://github.com/rustwasm/wasm-bindgen/issues/201)) you shouldn't need to use `pub` on the methods themselves